modules/ROOT/pages/_partials/_define_component_attributes.adoc

```diff
@@ -18,11 +22,20 @@ ifdef::releasetag[:tag: -{releasetag}]
 // version-maintenance is the full version including maintenance release number.
 // Note that the maintenance release is set at module level in the `_define_module_attributes.adoc` file
 // here at component level it is always same as version-full
-:version-maintenance-android: {major}.{minor}.{maintenance-android}
-:version-maintenance-c: {major}.{minor}.{maintenance-c}
-:version-maintenance-net: {major}.{minor}.{maintenance-net}
-:version-maintenance-java: {major}.{minor}.{maintenance-java}
-:version-maintenance-ios: {major}.{minor}.{maintenance-ios}
+:version-maintenance-android: {major}.{minor}.{maintenance-android}{tag}
```

Comment:
So, I don't know why these didn't *already* have {tag}...
but as this is a new addition, we'll need to test that this hasn't broken some behavious elsewhere.

Reply:
It's only used in install, the definition itself and quickstart. Have tested and verified output is correct in all locations.